devicetypes/smartthings/everspring-motion-sensor.src/everspring-motion-sensor.groovy

```diff
@@ -0,0 +1,426 @@
+/**
+ *  Copyright 2020 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Everspring Motion Sensor
+ *
+ *  Author: SmartThings
+ */
+
+metadata {
+	definition (name: "Everspring Motion Sensor", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.sensor.motion", runLocally: false, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false, genericHandler: "Z-Wave") {
+			capability "Motion Sensor"
+			capability "Battery"
+			capability "Health Check"
+			capability "Tamper Alert"
+
+			//RAW: zw:Ss type:0701 mfr:0060 prod:0001 model:0006 ver:1.00 zwv:4.61 lib:03 cc:5E,55,98 sec:86,85,59,72,5A,73,6C,7A,80,84,71,70 role:06 ff:8C07 ui:8C07                                                           
+			fingerprint mfr: "0060", prod: "0001", model: "0006", deviceJoinName: "Everspring PIR Sensor" 
```

Comment:
This fingerprint already exists in `zwave-motion-sensor` under ICP-11023. We're not going to go back and create new DTHs for already-certified devices.

Reply:
@greens  According to ICP-12456 I will close this PR and add preferences to existing DTH.